tools/flow/keys.py

```diff
@@ -130,7 +130,8 @@ def gen(self, name, ca):
         myname = x509.Name([
             x509.NameAttribute(NameOID.COMMON_NAME, name),
             x509.NameAttribute(NameOID.ORGANIZATION_NAME, u'Linaro Ltd'),
-            x509.NameAttribute(NameOID.ORGANIZATIONAL_UNIT_NAME, u'LinaroCA Device Cert - Signing'),
+            x509.NameAttribute(NameOID.ORGANIZATIONAL_UNIT_NAME,
+                               u'LinaroCA Device Cert - ' + purpose),
```

Comment:
Didn’t we want to get away from ‘LinaroCA’?

Reply:
Yeah, this should probably be something that indicates its "demo" nature.

Also, what was the reason we put the "Signing" part in the OU, but not in the purpose bits of the cert? Or have we just not gotten around to that?

Reply:
That sounds like a mistake or oversight.